react-example/src/views/Commerce.tsx

```diff
@@ -82,12 +132,12 @@ export const Commerce: FC<Props> = () => {
       <Heading>POST /trackPurchase</Heading>
       <EndpointWrapper>
         <Form onSubmit={handleTrackPurchase} data-qa-purchase-submit>
-          <label htmlFor="item-2">Enter Item Name</label>
+          <label htmlFor="item-2">Enter valid JSON</label>
           <TextField
             value={purchaseItem}
             onChange={(e) => setPurchaseItem(e.target.value)}
             id="item-2"
-            placeholder="e.g. keyboard"
+            placeholder='e.g. {"items":[{"id":"fdsafds","price":100}]}'
```

Comment:
same here. we need to pull in a json editor if we want to actually allow editing of that type.

Reply:
Done